jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -52,15 +52,20 @@ class Orientation(PluginTemplateMixin, ViewerSelectMixin):
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.close_in_tray`
     * ``align_by`` (`~jdaviz.core.template_mixin.SelectPluginComponent`)
     * ``wcs_fast_approximation``
-    * ``delete_subsets``
+    * :meth:`delete_subsets`
     * ``viewer``
     * ``orientation``
     * ``rotation_angle``
     * ``east_left``
-    * ``add_orientation``
+    * :meth:`add_orientation`
+    * :meth:`create_north_up_east_left`
+    * :meth:`create_north_up_east_right`

```

Comment:
should we remove `create_` from the method names since these can create AND/OR set?

Reply:
I might vote for `set_` rather than `create_`, since `set_` tells you the end result?

Reply:
unless you pass `set_as_orientation = False` 🤔... but I think I agree that I still prefer that